### PR TITLE
Created JSON for 2022/23 collection

### PR DESCRIPTION
**Pull request by roobottom**
Copied `2021_2022.json` and made the following changes:

## Manual changes to match `2021_2022.json`
* Manually copied across [changed from PR 496](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/508#issue-1215780317)
* Manually copied across [changes from PR 505](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/505/files)
* Manually copied across [changes from PR 512](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/512/files)
* Manually copied across [changes from PR 518](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/518/files)
* Manually copied across [changes from PR 513](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/513/files)

## Tenancy information
* Added new question "Is this a joint tenancy?" with the variable name `joint_tenancy`
* Updated response options on "What is the type of tenancy?" `tenancy`
* Updated response options on "What is the type of tenancy after the starter period has ended?" `tenancy`
* Updated depends-on values on "Length of fixed-term tenancy" `tenancylength`
* Updated response options on "Is this letting in sheltered accommodation?" `shelteredaccom`

## Household characteristics 
* Updated response options for "What is the lead tenant’s nationality?" `national`

## Household situation
* Removed question "Was the reason for leaving a direct result of the removal of the spare room subsidy or benefit cap introduced from 2013?" `underoccupation_benefitcap`
* Updated response options on "Did the household experience homelessness immediately before this letting?" `homelessness`

## Income, benefits and outgoings
* Updated response options for "Is the household likely to be receiving any of these housing-related benefits?" `hb`
* Updated depends-on values for "After the household has received any housing-related benefits, will they still need to pay for rent and charges?" `hbrentshortfall`

### CODE REVIEW

File: config/forms/2022_2023.json [hunk 1]
```diff
@@ -0,0 +1,5773 @@
...
+        },
+        "tenancy_information": {
+          "label": "Tenancy information",
+          "depends_on": [
+            {
+              "setup": "completed"
+            }
+          ],
+          "pages": {
+            "joint_tenancy": {
```
Comment (smeny) [inline config/forms/2022_2023.json]: Let's call this `joint`

Comment (smeny) [reply config/forms/2022_2023.json]: (to match CDS field)

**Comment by paulrobertlloyd** [comment]
FYI: #512 – Update to the privacy notice question (won’t need to add the partial, but will need to update the JSON for this question and that for the question group)

File: config/forms/2022_2023.json [hunk 2]
```diff
@@ -0,0 +1,5773 @@
...
+              "header": "",
+              "description": "",
+              "questions": {
+                "hbrentshortfall": {
+                  "check_answer_label": "Outstanding amount for basic rent and charges",
+                  "header": "After the household has received any housing-related benefits, will they still need to pay for rent and charges?",
+                  "hint_text": "Also known as the ‘outstanding amount’.",
+                  "type": "radio",
+                  "answer_options": {
+                    "0": {
```
Comment (smeny) [inline config/forms/2022_2023.json]: This should be 1 (Yes) / 2 (No) / 3 (DK)
I will fix this in 2021-2022

**Comment by paulrobertlloyd** [comment]
FYI: #518 - Including guidance text on both household income questions

**Review by paulrobertlloyd — APPROVED** [review]

**Pull request merged**